src/main/java/spring/boot/task/EmailService.java

```diff
@@ -0,0 +1,43 @@
+package spring.boot.task;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.stereotype.Service;
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+@Service
+public class EmailService {
+    private EmailConsoleReader emailConsoleReader;
+    private String fileName = "email";
```

Comment:
It would be better to move this into environment property(application.properties) cause it depends on the environment where this app executes (operating system, etc.)